src/uct/sm/cma/cma_iface.c

```diff
@@ -61,7 +61,7 @@ static ucs_status_t uct_cma_iface_query(uct_iface_h tl_iface,
                                           UCT_IFACE_FLAG_CONNECT_TO_IFACE;
     iface_attr->latency.overhead        = 80e-9; /* 80 ns */
     iface_attr->latency.growth          = 0;
-    iface_attr->bandwidth               = 12000 * 1024.0 * 1024.0;
+    iface_attr->bandwidth               = 11687 * 1024.0 * 1024.0;
```

Comment:
should be 11687*10e6 (osu reports power of 10)

Reply:
really?
ok, re-calculated to power of 2